### PR TITLE
Add Battlefield V

### PR DESCRIPTION
**Pull request by EyItchy**

### CODE REVIEW

File: customizations/Battlefield™ V.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "color": "#0384fc",
+    "imageUrl": "https://i.imgur.com/9NJho2u.jpg",
```
Comment (MuteBefehl) [inline customizations/Battlefield™ V.json:3]: Please use an image without text.
Also, the color does not fit.

**Review by MuteBefehl — CHANGES_REQUESTED** [review]

File: customizations/Battlefield™ V.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "color": "#0384fc",
```
Comment (vascYT) [inline customizations/Battlefield™ V.json:2]: Please use a different color

File: customizations/Battlefield™ V.json (hunk 1 above)
Comment (vascYT) [inline customizations/Battlefield™ V.json:3]: Also use an image without text.

**Comment by MuteBefehl** [comment]
As privately agreed, I close this pr because the requested changes were not made.

**Pull request closed without merging**